PHPCI/Migrations/20150324174958_fix_email_and_name_user_fields.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class FixEmailAndNameUserFields extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $user_table = $this->table('user');
+        $user_table
+            ->addIndex('email', ['unique' => true])
+            ->addIndex('name', ['unique' => true])
+            ->save();
+    }
+
+    /**
+     * Migrate Down.
+     */
+    public function down()
+    {
+        $user_table = $this->table('user');
+        $user_table
+            ->removeIndex('email', ['unique' => true])
+            ->removeIndex('name', ['unique' => true])
+            ->save();
+    }
+}
```

Comment:
I prefer to add an empty line to the end of the files, but I'm not sure CS require it...